pkg/service/accounts.go

```diff
@@ -24,7 +24,7 @@ func (s *AccountsService) AddAccount(accountID, adminRoleARN string) {
 		},
 	}
 	params.SetTimeout(5 * time.Second)
-	_, err := apiClient.PostAccounts(params, nil)
+	_, err := ApiClient.PostAccounts(params, nil)
```

Comment:
Needed to make these public, so I can mock them in tests
(our lease service relies on global variables)